jquery.flot.time.js

```diff
@@ -278,11 +280,21 @@ API.txt for details.
     var specQuarters = baseSpec.concat([[1, "quarter"], [2, "quarter"],
         [1, "year"]]);
 
+    var axisFirstData; //record first value of the datapoints
```

Comment:
I'd like you to spend some time looking at how the relative time works in LV NXG, LV CG and think about it, then add some documentation.

The code that I wrote for relative time is a mess because I didn't spent that time to research and understand relative time.